docs/_static/custom.js

```diff
@@ -0,0 +1,36 @@
+function getSelectedDocsVersion(pathname) {
+    if (!pathname) {
+        pathname = document.location.pathname
+    }
+    let parts = pathname.split('/').filter(item => item !== "");
+    if (parts.length === 1) {
+        if (parts[0] === PROJECT) {
+            // '/ape/' (return 'stable')
```

Comment:
This comments might be misleading, not sure the correct path but I think it's `ape-starknet`.

```suggestion
            // '/ape-starknet/' (return 'stable')
```